src/viam/rpc/dial.py

```diff
@@ -30,7 +30,7 @@ class Credentials:
     Currently only supports robot location secret.
```

Comment:
[minor] looks like we can remove this line. Yay!

Reply:
updated